modules/account-lib/test/unit/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -196,8 +196,11 @@ describe('TSS ECDSA TESTS', function () {
 
       config = [
         { signerOne: A, signerTwo: B },
+        { signerOne: A, signerTwo: C },
```

Comment:
used to help generate new fixtures. Also ensures we test all combinations of 2/3 signing between parties a, b, c.